build.gradle

```diff
@@ -141,3 +143,18 @@ task unitTest(type: Test) {
     outputs.upToDateWhen { false }
 }
 
+def googleCredentialsFile = "${projectDir}/src/test/resources/rendered/sa-account.json"
+
+task integrationTest(type: Test) {
+    // Sets the application Google Credentials.
+    environment = [
+            'GOOGLE_APPLICATION_CREDENTIALS': "${googleCredentialsFile}"
```

Comment:
We're going to need the application google credentials set for running the service as well. Can we start a service locally or in k8s with these changes as-is, or do we need to add environment variables to run_local.sh and the deployment.yaml?
Should running locally call render-config.sh?

Reply:
Local: we will need to modify the run_local.sh to set system environment variables.  I didn't put it because we turned off the pubsub in local server and we don't need this yet. 

K8s: I further think we should also add this into helm-chart it in k8s deployment to tell the service use the sa account. 
That is what I did for IC:
https://github.com/broadinstitute/terra-helm/blob/master/charts/identityconcentrator/templates/deployment.yaml#L35
Also, I checked the GKE log seems it is using the correct SA account somehow. But as we are adding more SA accounts and make them granular, we should have `GOOGLE_APPLICATION_CREDENTIALS ` in helm-chart.